appveyor.yml

```diff
@@ -0,0 +1,30 @@
+version: '{build}'
+
+pull_requests:
+  do_not_increment_build_number: true
+
+skip_tags: true
+
+image:
+- Visual Studio 2019
+
+configuration:
+- Release
+- Debug
+
+platform:
+- x64
+#- x86 # by design PS4Delta can't support x86 systems
+
+install:
+- cmd: |
```

Comment:
cmd is the default shell so you can just do `- |`

Reply:
Thx for the tip. Done. Works well:
https://ci.appveyor.com/project/q4a/ps4delta/builds/30503622